documentation/asciidoc/computers/raspberry-pi/display-parallel-interface.adoc

```diff
@@ -335,162 +335,115 @@ h|*27* h|*26* h|*25* h|*24* h|*23* h|*22* h|*21* h|*20* h|*19* h|*18* h|*17* h|*
 
 === Disable Other GPIO Peripherals
 
-Note that all other peripheral overlays that use conflicting GPIO pins must be disabled. In config.txt, take care to comment out or invert any dtparams that enable I2C or SPI:
+Note that all other peripheral overlays that use conflicting GPIO pins must be disabled. In `config.txt`, take care to comment out or invert any dtparams that enable I2C or SPI:
```

Comment:
Please don't start a sentence with "Note" unless you're going to make a `NOTE:` block.